mec/CMakeLists.txt

```diff
@@ -4,7 +4,5 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-zephyr_include_directories_ifdef(CONFIG_SOC_SERIES_MEC1701X .)
```

Comment:
this one not needed?

Reply:
Apparently... there's no reference to `MEC1701X` that I could find and all boards build fine.